File: Task 03/ruby/Subtask3/Task3.rb
print "Enter the number of rows for diamond: "
n = gets.chomp.to_i

(0...n).each do |i|
  print ' ' * (n - i - 1)
  print '*' * (2 * i + 1)
  puts
end

(n - 2).downto(0) do |i|
  print ' ' * (n - i - 1)
  print '*' * (2 * i + 1)
  puts
end

File: Task 03/elixir/Subtask3/Task3.ex
n=IO.gets("Enter width of diamond") |> String.trim() |> String.to_integer()
for i<-0..(div(n,2))do
	spaces=div(n-2*i-1,2)
	ast=2*i+1
	IO.puts(String.duplicate(" ", spaces) <> String.duplicate("*",ast))
     end
for i<- (div(n,2) -1)..0 do
	spaces=div(n-2*i-1,2)
	stars=2*i+1
	IO.puts(String.duplicate(" ",spaces)<> String.duplicate("*", ast))
     end
end
